pkg/portal/middleware/aad_test.go

```diff
@@ -284,6 +286,7 @@ func TestCheckAuthentication(t *testing.T) {
 }
 
 func TestLogin(t *testing.T) {
+
```

Comment:
nit: no new line at the top of a function.

Reply:
Resolved, thanks